test/markdown_renderers.js

```diff
@@ -28,4 +30,22 @@ describe('apifyMarked custom renderer works', () => {
         expect(renderedTitle).to.equal(`
             <h3 id="welcome-title-id"><a href="#welcome-title-id"></a>Welcome to Apify</h3>`);
     });
+
+    it('replaces relative URLs from GitHub repos with absolute URLs in links', () => {
+        const href = './src/foo/bar';
+        const text = 'link to bar';
+        const repoUrl = `https://github.com/${repoFullName}`;
+        const renderedLink = customLinkRenderer(href, text, repoUrl, branchName);
+
+        expect(renderedLink).to.equal('<a href=https://github.com/apify/actor-test-url/tree/main/src/foo/bar rel="nofollow noreferrer noopener">link to bar</a>')
```

Comment:
href=https://github.com/apify/actor-test-url/tree/main/src/foo/bar

Reply:
Sorry, what do you mean with this one?